tools/visual-pipeline-and-platform-evaluation-tool/Dockerfile.vippet

```diff
@@ -14,7 +14,7 @@
 
 # SPDX-License-Identifier: Apache-2.0
 
-FROM docker.io/intel/dlstreamer:2025.0.1.3-ubuntu24
+FROM ghcr.io/open-edge-platform/edge-ai-libraries/dls-deb-ubuntu24:latest

```

Comment:
This image is not available without logging to `ghcr` registry. 
I don't think we want to have this approach in `main` branch.